ieeeuoft/makeuoft-site-2021
# Remove User-facing Application Team Logic

## Overview

- Resolves #10
- Removes the team section from the dashboard
- Removes the leave team view
- Removes the leave team form and associated dashboard logic



## Unit Tests Created

- Removes all tests related to the removed functionality, no tests added


## Steps to QA

- Update the registration open date in the settings file so you can apply, apply, and fill out an application. Make sure the dashboard looks proper afterwards (nothing about teams). On the admin site, the application should still be associated with a new random team.
